conans/test/update_settings_yml_test.py

```diff
@@ -0,0 +1,71 @@
+import unittest
+from conans.test.tools import TestClient
+from conans.util.files import load, save
+
+
+class UpdateSettingsYmlTest(unittest.TestCase):
+    """ This test is to validate that after adding a new settings, that allows a None
```

Comment:
Yeah, maybe useless, but it test this new behavior using the build_type. If we are keeping the new None behavior with shas should keep the test.